build_projects/shared-build-targets-utils/DependencyVersions.cs

```diff
@@ -7,7 +7,9 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class DependencyVersions
     {
-        public static readonly string CoreCLRVersion = "1.0.2";
-        public static readonly string JitVersion = "1.0.2";
+        // TODO: Update these for consuming servicing version of CoreCLR packages
+        //       Also, update CoreCLR package referenced at /Users/gkhanna/Github/gkhanna79/core-setup/pkg/projects/Microsoft.NETCore.App/project.json
```

Comment:
@ellismg PTAL at the 3rd commit that adds entry for OSX 10.12 validation and also enable building host packages for 1.0.2.

CC @weshaggard